core/network/types/status.hpp

```diff
@@ -101,8 +102,9 @@ namespace kagome::network {
   template <class Stream,
             typename = std::enable_if_t<Stream::is_decoder_stream>>
   Stream &operator>>(Stream &s, Status &v) {
-    return s >> v.version >> v.min_supported_version >> v.roles >> v.best_number
-           >> v.best_hash >> v.genesis_hash >> v.chain_status;
+    uint8_t _;
```

Comment:
Please comment.

Reply:
It's a temporary solution before refactoring of networking. ```Status``` contains unsigned 1 byte value 0 as the first byte of serialized data. In contrast with the existed messages ```data``` is not scale encoded into buffer before encoding the whole message. This field and this kind of encoding are legacy now and should be reworked.